common/core/gpudevice.cpp

```diff
@@ -518,6 +560,50 @@ void GpuDevice::HandleHWCSettings() {
     temp_displays.swap(total_displays_);
   }
 
+  // Now set floating display configuration
+  // Get the floating display index and the respective rectangle
+  // TODO Logical display on & mosaic display on scenario
+  if (use_float && !use_logical && !use_mosaic) {
+    size_t size = float_display_indices.size();
+    bool ret = false;
+
+    // Set custom resolution to desired display instance
+    for (size_t i = 0; i < size; i++) {
+      std::vector<int32_t> &float_rect = float_displays.at(i);
+      HwcRect<int32_t> rect = HwcRect<int32_t>(float_rect.at(0),
```

Comment:
Instead of here, you can do it line 367 above ?

Reply:
total displays won't be updated at 370. Both ordering and physical connectivity of displays happen after 400.

Reply:
Sorry, I mean instead of converting vector to rect, you could do that at line 370

Reply:
Oh yes, that can be done. I will do that.